### PR TITLE
109 frameworks

### PR DESCRIPTION
**Pull request by SiteSeerBE**
- Lots of changes to bring competencies into frameworks.
- Several test scripts still have to be refactored.
Ref: #109

### CODE REVIEW

**Comment by SiteSeerBE** [comment]
Volgens testing kan competencyFramework null zijn. Kunnen we dat non-nullable maken?

**Review by shkurata — APPROVED** [review]
:+1:

**Comment by Moeriki** [comment]
FYI Ik wacht met review tot de testen slagen.

**Comment by SiteSeerBE** [comment]
> FYI Ik wacht met review tot de testen slagen.

Noted & Resolved.

Sorry van de 💩. Ik had eerst gewoon een assert toegevoegd, maar dacht dan net voor het comitten aan een betere oplossing.

File: app/src/pages/manage/frameworks/[frameworkId]/[competencyId]/index.vue [hunk 1]
```diff
@@ -16,7 +16,8 @@ const { t } = useI18n();
 const router = useRouter();
 
 const props = defineProps<{
-  id: string; // route param
+  competencyId: string;
+  frameworkId: string;
```
Comment (Moeriki) [inline app/src/pages/manage/frameworks/[frameworkId]/[competencyId]/index.vue:20]: Inconsistent usage of `// route param`. I don't mind if you keep using it or not. Remove everywhere or add everywhere relevant.

**Review by Moeriki — CHANGES_REQUESTED** [review]
A handful of tests were deleted. Was this intentional?

File: app/src/pages/manage/frameworks/[frameworkId]/[competencyId]/index.vue (hunk 1 above)
Comment (SiteSeerBE) [reply app/src/pages/manage/frameworks/[frameworkId]/[competencyId]/index.vue:20]: Removed everywhere.

**Review by Moeriki — APPROVED** [review]

**Comment by SiteSeerBE** [comment]
> A handful of tests were deleted. Was this intentional?

Yes, although it was intentional, I now see they weren't that hard to refactor to the new structure. Some I just had to move in another folder :)

**Review by Moeriki — APPROVED** [review]

**Pull request merged**